### PR TITLE
Deploy injective and inEVM

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

- Deploy core to injective and inEVM networks

### Drive-by changes

- Update cw-hyperlane dependency to https://github.com/hyperlane-xyz/cosmwasm/releases/tag/v0.0.6

### Related issues

- 

### Backward compatibility

Yes

### Testing

- Fork tests
- e2e tests

### CODE REVIEW

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:456]: Short is optional if the name is already pretty short

File: typescript/sdk/src/consts/chains.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chains.ts:73]: is inEVM missing here?

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:464]: Chains chains also need a `restURLs` field for the warp UI to work right (see neutron)

File: typescript/infra/scripts/deploy.ts
Comment (jmrossy) [inline typescript/infra/scripts/deploy.ts:83]: Strange to see an address hardcoded in here

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:424]: Avoid tailing slashes please for consistency

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:468]: ditto

File: typescript/sdk/src/consts/environments/mainnet.json
Comment (jmrossy) [inline typescript/sdk/src/consts/environments/mainnet.json:231]: testRecipient & testTokenRecipient?
Looks like they're missing for manta above too

File: typescript/sdk/src/metadata/chainMetadataTypes.ts
Comment (jmrossy) [inline typescript/sdk/src/metadata/chainMetadataTypes.ts:248]: `is*` functions like this shouldn't throw IMO, or the return value is never false

File: typescript/sdk/src/core/MultiProtocolCore.ts
Comment (jmrossy) [inline typescript/sdk/src/core/MultiProtocolCore.ts:40]: Y dis?

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [reply typescript/sdk/src/consts/chainMetadata.ts:468]: Also I recommend adding more RPCs to take advantage of SmartProvider's fallback

File: typescript/sdk/src/consts/chainMetadata.ts
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:443]: I recommend adding more RPCs to take advantage of SmartProvider's fallback:
https://github.com/cosmos/chain-registry/blob/master/injective/chain.json#L217

File: rust/config/mainnet3_config.json
Comment (nambrot) [inline rust/config/mainnet3_config.json:863]: was this intentional?

File: rust/utils/run-locally/Cargo.toml
Comment (nambrot) [inline rust/utils/run-locally/Cargo.toml:36]: ditto

File: rust/utils/run-locally/src/cosmos/mod.rs
Comment (nambrot) [inline rust/utils/run-locally/src/cosmos/mod.rs:61]: Is that our versioning scheme now?

File: typescript/infra/config/environments/testnet4/validators.ts
Comment (nambrot) [inline typescript/infra/config/environments/testnet4/validators.ts:292]: Stale?

File: typescript/infra/config/environments/testnet4/warp/addresses.json
Comment (nambrot) [inline typescript/infra/config/environments/testnet4/warp/addresses.json:1]: Should this be added?

**Review by nambrot — APPROVED** [review]

File: rust/utils/run-locally/src/cosmos/mod.rs
Comment (yorhodes) [reply rust/utils/run-locally/src/cosmos/mod.rs:61]: idk what the other one was :P
this made it easier to follow

File: typescript/infra/config/environments/testnet4/validators.ts
Comment (yorhodes) [reply typescript/infra/config/environments/testnet4/validators.ts:292]: will wait until @tkporter 's PR gets merged

**Pull request merged**
